leetcode/intervals.md

```diff
@@ -0,0 +1 @@
+# Intervals
```

Comment:
Your Pull Request title: LEETCODE-1022 Remove nth node from end of list

Check result:

* Pull Request title action must start with ['Added', 'Deleted', 'Refactored', 'Moved', 'Fixed']